C/guide/ch_budgets.xml

```diff
@@ -206,7 +206,7 @@
 
       <para>The budgeted balance sheet is similar to the balance sheet. Both show the assets, liabilities, and
         equity. The difference is that the balance sheet is based on historical data, and the
-        <emphasis>budgeted</emphasis> balance sheet is based on the predictions made in the budget.
+        <emphasis>budgeted</emphasis> balance sheet is based on the predictions made in the budget.  For more details, see <ulink url="&url-docs-C;manual/report-classes.html#report-budget"><quote>Budget Reports</quote> in the manual</ulink>
```

Comment:
IMHO most of the `<sect1 id="budget_reporting1">` is still descriptive and belongs into the manual. Here I would expect rules to get the best results  from using the reports like e.g. `At first use report x to check for input errors. `